### PR TITLE
🌐🇵🇱 Add Polish translation for `docs/pl/docs/tutorial/first-steps.md`

### PR DESCRIPTION
**Pull request by Valaraucoo**
Created `docs/pl/tutorial/first-steps.md` 🇵🇱 Polish translation.

### CODE REVIEW

**Review by dominikwozniak — APPROVED** [review]

File: docs/pl/docs/tutorial/first-steps.md [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+
+Zobaczysz alternatywną automatycznie wygenerowaną dokumentację API (dostarczoną przez <a href="https://github.com/Rebilly/ReDoc" class="external-link" target="_blank">ReDoc</a>):
+
+![ReDoc](https://fastapi.tiangolo.com/img/index/index-02-redoc-simple.png)
+
+### OpenAPI
+
+**FastAPI** generuje "schemat" z całym Twoim API przy użyciu standardu **OpenAPI** do definiowania API.
+
+#### "Schemat"
```
Comment (BartoszCki) [inline docs/pl/docs/tutorial/first-steps.md:72]: IMHO some keywords, including word "schema", should not be translated

File: docs/pl/docs/tutorial/first-steps.md [hunk 2]
```diff
@@ -0,0 +1,334 @@
...
+
+#### "Schemat" danych
+
+Termin "schemat" może również odnosić się do wyglądu niektórych danych, takich jak zawartość JSON.
+
+W takim przypadku będzie to oznaczać atrybuty JSON, ich typy danych itp.
+
+#### OpenAPI i JSON Schema
+
+OpenAPI definiuje schemat API dla Twojego API. A ten schemat zawiera definicje (lub "schematy") danych wysyłanych i odbieranych przez Twój interfejs API przy użyciu **Schematu JSON**, standardu dla schematów danych JSON.
```
Comment (BartoszCki) [inline docs/pl/docs/tutorial/first-steps.md:90]: Translating  "JSON Schema" to "schemat JSON" makes no sense as it (JSON Schema) is a proper noun

File: docs/pl/docs/tutorial/first-steps.md [hunk 3]
```diff
@@ -0,0 +1,334 @@
...
+```
+
+...ścieżką będzie:
+
+```
+/items/foo
+```
+
+!!! info
+    "Ścieżka" jest zazwyczaj nazywana "endpoint" lub "route'.
```
Comment (BartoszCki) [inline docs/pl/docs/tutorial/first-steps.md:204]: How  about adding "path"? Like:
"Ścieżka" jest zazwyczaj nazywana "path", "endpoint" lub "route'.

File: docs/pl/docs/tutorial/first-steps.md [hunk 4]
```diff
@@ -0,0 +1,334 @@
...
+
+Podczas budowania API, "ścieżka" jest głównym sposobem na oddzielenie "troski" i „zasobów”.
```
Comment (BartoszCki) [inline docs/pl/docs/tutorial/first-steps.md:206]: I'm not sure what's a better translation but "troski" is definitely not the word ;)

**Review by BartoszCki — CHANGES_REQUESTED** [review]
In general it's OK but there are several places when translations don't feel natural

**Review by BartoszCki — APPROVED** [review]
Ship it!

**Comment by tiangolo** [comment]
Awesome, thank you @Valaraucoo! :bow: 

And thanks for the reviews @BartoszCki and @dominikwozniak ! :cake:

**Pull request merged**